tests/testproject/settings.py

```diff
@@ -42,17 +42,20 @@
     'django_any',
 ]
 
-MIDDLEWARE_CLASSES = [
+MIDDLEWARE = MIDDLEWARE_CLASSES = [
```

Comment:
I think we shouldn't assign MIDDLEWARE_CLASSES var for Django >= 1.0, we can build full list of middleware classes and then assign them to correct variable name based on current Django version

Reply:
MIDDLEWARE_CLASSES will be ignored anyway. Why we need to duplicate code? When 1.8 will be dropped we will simply remove this assignment.

Reply:
I don't think it would too much duplication:
```
middlewares = [
    'django.contrib.sessions.middleware.SessionMiddleware',
    'django.middleware.common.CommonMiddleware',
    'django.middleware.csrf.CsrfViewMiddleware',
    'django.contrib.auth.middleware.AuthenticationMiddleware',
    'django.contrib.messages.middleware.MessageMiddleware',
    'django.middleware.clickjacking.XFrameOptionsMiddleware',
]
if django.VERSION >= (1, 8):
    middlewares.append('django.middleware.security.SecurityMiddleware')
if django.VERSION <= (1, 9):
    middlewares.append('django.contrib.auth.middleware.SessionAuthenticationMiddleware')
if django >= (1, 10):
    MIDDLEWARE = middlewares
else:
    MIDDLEWARE_CLASSES = middlewares
```

Reply:
Ok, as you wish :)